src/jade/tabs/nzd.jade

```diff
@@ -0,0 +1,109 @@
+              | Coinex (~Coinex)
+              br
+              a(href="https://coinexgateway.com", target="_blank") www.coinexgateway.com
+        .col-md-3
+            .title(l10n) Availability
+            .description(l10n) Australia and New Zealand
+        .col-md-3
+            .title(l10n) Fees
+            .description(l10n) 
+              p(l10n) .98% for deposits and withdrawals&#32;
+              p(l10n)
+                a(href="https://coinexgateway.com/Gateway/Fees.html", target="_blank") More details 
+        .col-md-3
+            .title(l10n) Rating
+            .description(l10n)
+              a(href="https://coinist.co/ripple/gateways/coinex", target="_blank") read reviews
+
+      .row.fund-tab-description
+        .col-xs-12.col-md-8.col-lg-6
+          .descriptor(ng-show="!nzdConnected", l10n) Ripple Trade has partnered with Ripple New Zealand to provide easier access to NZD. Following this action will enable Ripple New Zealand to hold NZD on your behalf.
+
+      .row.fund-tab-description
+        .col-xs-7.col-sm-5.col-md-4
+          button.btn.btn-large(ng-show="!nzdConnected && !loading", type="submit", ng-click="save_account()", ng-disabled="!account.Balance") Add NZD Coinex
+          button.btn.btn-large(ng-show="!nzdConnected && loading", type="submit", ng-disabled="loading") Adding...
+          button.btn.btn-large(ng-show="showInstructions && nzdConnected", type="submit", ng-click="toggle_instructions()", l10n) Hide instructions
+          button.btn.btn-large(ng-show="nzdConnected && !showInstructions", type="submit", ng-click="toggle_instructions()", l10n) Show instructions
+
+        .connect.col-xs-5.col-sm-7.col-md-8(ng-hide="loading")
+          .description(ng-show="!nzdConnected && account.Balance", l10n) 
+            i.fa.fa-times
+            |  Not connected
+          .description(ng-show="nzdConnected && account.Balance", l10n)
+            i.fa.fa-check
+            |  Connected
+          .description(ng-show="!account.Balance", l10n) Your account has to be activated before you can add a gateway account.
+
+      .row(ng-show="showInstructions && nzdConnected")
+        .instructions.col-md-10
+          a.dismiss(href="", id="hide", ng-click="toggle_instructions()", l10n)  ×
+          .title(l10n) To Initiate New Zealand Dollar Deposits and Withdrawals
+          .description(l10n) 
+            | Register or login to your Coinex account. You will need to submit proof of identity.&#32;
+            a(href="https://coinexgateway.com/Gateway/Verification.html", target="_blank") Details here.&#32;
+            
+          .title(l10n) Deposit Options
+          .description(l10n) 
+            | Coinex accepts deposits by the following methods:
+            ul 
+              li
+                p(l10n) International bank transfer
+              li
+                p(l10n) Ripple transaction
+              li
+                p(l10n) Stellar transaction
```

Comment:
yes, it's fine :)

On Thu, Jan 15, 2015 at 12:14 PM, Malika Rajvanshy <notifications@github.com
> wrote:

> In src/jade/tabs/nzd.jade
> <https://github.com/ripple/ripple-client/pull/2106#discussion-diff-23039824>
> :
>
> > +          a.dismiss(href="", id="hide", ng-click="toggle_instructions()", l10n)  ×
> > +          .title(l10n) To Initiate New Zealand Dollar Deposits and Withdrawals
> > +          .description(l10n)
> > +            | Register or login to your Coinex account. You will need to submit proof of identity.&#32;
> > +            a(href="https://coinexgateway.com/Gateway/Verification.html", target="_blank") Details here.&#32;
> > +
> > +          .title(l10n) Deposit Options
> > +          .description(l10n)
> > +            | Coinex accepts deposits by the following methods:
> > +            ul
> > +              li
> > +                p(l10n) International bank transfer
> > +              li
> > +                p(l10n) Ripple transaction
> > +              li
> > +                p(l10n) Stellar transaction
>
> Do we want to advertise Stellar?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ripple/ripple-client/pull/2106/files#r23039824>.
>